### ISSUE TITLE
Contiguous seeds for iterations > 1

### ISSUE THREAD

**Issue by JohannesGaessler** [invoke-ai/InvokeAI#909]
**Is your feature request related to a problem? Please describe.**
I did a [statistical analysis](https://github.com/JohannesGaessler/stable-diffusion-insights) regarding the speed of deterministic k-Diffusion samplers.
With the code I was using there were technical problems when setting the number of steps >500 for DDIM and PLMS.
This repository seems to be able to generate images with steps > 500 for those samplers without issue.
However, unfortunately there does not seem to be a way to set the seeds used for generation to a contiguous range like 0-89 when generating more than one image at once.
I would appreciate the option to enable this since a proper statistical analysis requires reproducibility and it's much easier to specify a range of seeds for the images that were used than it is to specify a list of seemingly random seeds.

**Describe the solution you'd like**
An option that explicitly sets a seed at each iteration, starting at the user-specified seed and then being incremented by 1 at each iteration.
The result should be that for example with --seed 0 and --iterations 20 images for the seeds 0-19 are generated.

**Describe alternatives you've considered**
I could manually run the generation of images for the desired seeds but that would be very tedious.
It's also problematic in situations where I will not interact with the machine that generates the images for an extended period of time.

**Comment by lstein** [comment]
I'm happy to hear about your research and help support it. There's a single function that sets the random seed each time a new image is generated, and this could easily be altered to simply increment the previous seed by one as a user-settable option.

But before I make this change, I want to confirm with you that the current behavior is not usable. The "random" seeds follow a deterministic order. If you specify a fixed seed at the beginning of a series of generations (e.g. using -n100), you'll get the same series of random seed each time. The seeds are logged and also written into the metadata of the PNG files.

**Comment by JohannesGaessler** [comment]
Perhaps I communicated my position poorly: I have already generated data for the seeds 0-89 for all of the deterministic samplers.
I cannot simply use different seeds since then the data would not be comparable.
(Setting the seed to 0 seems to result in a random seed for this repository but discarding only 1/90 of the samples is acceptable.)

When generating a completely new dataset the current implementation could be used for my purposes, but it would still be inconvenient.
It's true that as long as the sequence of seeds is always the same I could use an arbitrary sequence.
But if seeds aren't explicitly being set this behavior is very inconvenient with incremental data generation.
Generally I will not know a priori how much data I will need for sufficiently precise results.
Typically I will therefore generate some data first, run the analysis, and then decide how much data I still need to generate.
The problem with the current implementation is that I will not be able to easily determine which seed comes after the last seed that I used previously.
I'd have to generate one extra sample by starting at the last known seed and then carefully make sure to delete the extra samples afterwards to correctly continue the sequence.
If I were to instead combine samples from multiple sequences that in turn becomes inconvenient if I decide to add additional datapoints since I would no longer be able to generate all of the samples that I need in one go.

Another issue is validation: I can very easily check manually or via script whether the samples that I generated have a contiguous range of seeds.
It's more work to check whether I have the exact same seeds for each datapoint.
